core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+.fa {
```

Comment:
This is defined too broadly -- it hits all the other .fa classes in the application, not just these ones. (Please grep for the locations of the others and verify that they are unaffected before submitting the PR.)

Also, why do you need it? If you need to style the elements in the footer then add a custom class instead that is prefixed with "oppia-" so we know it's one of ours.